src/shapes/group.class.js

```diff
@@ -396,10 +396,15 @@
        */
       _render: function (ctx) {
         //  render fill/stroke courtesy of rect
-        //fabric.Rect.prototype._render.call(this, ctx);
+        this._hasFillStroke() && fabric.Rect.prototype._render.call(this, ctx);
         this._renderObjects(ctx);
       },
 
+      _hasFillStroke: function () {
+        return (this.fill && this.fill !== 'none' && this.fill !== 'transparent') ||
+          (this.stroke && this.stroke !== 'none' && this.stroke !== 'transparent' && this.strokeWidth);
+      },
```

Comment:
those exists has this.hasFill and this.hasStroke.
The 'none' value i m not sure if is reported somewhere, but shouldn't exist. I know it can come from SVG but we usually fiter it out